public/scss/components/_headings.scss

```diff
@@ -4,3 +4,7 @@ h1, h2, h3, h4, h5, h6 {
         padding-bottom: 0.5em;
     }
 }
+// Word-wrap the committee names when needed
+h1{
```

Comment:
Do we want this behavior for _all_ primary headings?

Reply:
Good point. I don't see any reasons for not doing it, but it is a bit too broad of a selector, maybe?

Reply:
Just to be safe, we could add a new class `.break-word` to the primary headings on the organ pages. This makes sure we don't break the styling on other pages.